_faqs/M1Workaround.md

```diff
@@ -2,34 +2,56 @@
 title: How to Install BciPy on Mac M1 Chip 
 ---
 
-If you are using a computer with the Mac M1 chip, you will run into issues with installing some packages of BciPy including numpy and psychopy. 
+If you are using a computer with the Mac M1 chip, you may run into issues with installing some packages of BciPy including psychopy and pyqt5. 
 
-The best workaround for this is to use the Rosetta Terminal. 
+The best workaround for this is to use the Rosetta 2 Terminal, which emulates an i386 architecture instead of arm64.
 
-## To create the Rosetta Terminal: 
-1. Duplicate the Apple terminal and rename one of the terminals "Rosetta Terminal" 
-2. Right click on the terminal you have renamed and click "Get info"
-From the "Get info" menu, check the box that says "Open using Rosetta" 
+## To create a Rosetta Terminal: 
+1. Open finder and naviagate into applications. Locate the Terminal app
 
-Launch the Rosetta Terminal and use only the Rosetta Terminal for all the following steps/any usage of BciPy. 
+2. Right click on the terminal app and click "Get info". From the "Get info" menu, check the box that says "Open using Rosetta".
+
+3. Force quit the terminal application and reopen a new terminal window.
+
+Verify you're in the rosetta terminal by running `arch` command (should see `i386`). Only use the Rosetta Terminal for all the following steps/any usage of BciPy. You can configure your IDE to use rosetta terminal too through some googling.
 
 
 ## To Install BciPy (within the Rosetta Terminal)
-1. Git clone https://github.com/BciPy/BciPy.git
+
+Pre-requisite: install [brew](https://docs.brew.sh/Installation)
+
+1. Git clone https://github.com/BciPy/BciPy.git. Check in to the most up to date branch.
+
 2. Run the following command in your terminal:
     > `arch -arm64 brew install python@3.9`
+
+    Reinstall If you already had python 3.9
+    > `arch -arm64 brew reinstall python@3.9`
+
 3. Create a virtual environment using python 3.9:
-    > `python3.9 -m venv venv`
+    > `python3 -m venv --system-site-packages venv`
 4. Open the virtual environment:
-    > `source venv/bin/activate` 
-5. Run the following command in your terminal (make sure that your pip is up to date if there are any issues at this step):
-    >  `pip install -e .` 
-6. Install XCode and enable command line tools:
+    > `source venv/bin/activate`
+
+5. Edit the following lines in `requirements.txt` accordingly
+- pylsl==1.14.0 --> pylsl==1.15.0
```

Comment:
I added a ticket to check if these can be updated across OS. It looks like the pyqt5 stuff will still remain, though!

Reply:
https://www.pivotaltracker.com/story/show/185927275